src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -362,6 +371,10 @@ public Response updateItem(UUID itemId, Item payload) throws Exception {
     String termsOfDelivery = payload.getTermsOfDelivery();
     Boolean allowPurchaseContactSeller = payload.getPaymentMethods().isAllowContactSeller();
     Boolean allowPurchaseCreditCard = payload.getPaymentMethods().isAllowCreditCard();
+    Boolean allowDelivery = payload.isAllowDelivery() != null ? payload.isAllowDelivery() : false;
+    Boolean allowPickup = payload.isAllowPickup() != null ? payload.isAllowPickup() : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=39&issues=AWXYx1xjlgbYnohYXER7&open=AWXYx1xjlgbYnohYXER7)

[ISSUE_KEY]: AWXYx1xjlgbYnohYXER7